tool/randomizer.lua

```diff
@@ -208,7 +217,7 @@ end
 function randomizer.reset()
     _G.onlevelchange = nil
     _G.onreset = nil
-    queue = {{seed = ""}, {seed = "", difficulty = difficulties[1]}}
+    queue = {{seed = ""}}
```

Comment:
Why are you deleting the second entry, when the queue is initially initialized with `queue = {{seed = ""}, {seedtype = randomizer.SEEDTYPE.KEEPSEED}}`? Don't you need the second element anymore for any other code which depends on it?

Reply:
yeah, a second element is not needed while the randomizer is not active. keeping queue a single element list before anything happens doesn't break anything and keeps everything clean. I've adapted all the necessary code (which was mainly `randomizer.hudlines()`) so that the program works as is with no errors.